Tests/Acceptance/CheckoutTestCase.php

```diff
@@ -178,4 +178,18 @@ public function assertPaymentSuccessful()
     {
         $this->assertTrue($this->isThankYouPage($this->getLocation()), 'Payment was not successful.');
     }
+
+    /**
+     * Checks if the given URL is pointing to the Masterpass redirect page.
+     *
+     * @param string $sUrl
+     *
+     * @return bool
+     */
+    public function isMasterpassRedirectPage($sUrl)
```

Comment:
Another approach would be to make a generic assertion for whether or not the user is redirected to an external  page.

Something like this (untested):

```php
public function assertRedirected()
{
    $this->waitForPageToLoad(self::WAIT_TIME_EXTERNAL * 1000); // `waitForPageToLoad` uses milliseconds
    $this->assertTrue(strpos($sUrl, $this->_getShopUrl()) === false, 'User was not redirected.');
}
```